core/src/epicli/data/common/ansible/playbooks/roles/vault/templates/script.config.j2

```diff
@@ -0,0 +1,6 @@
+VAULT_INSTALL_PATH="{{ specification.paths.main }}"
+ENABLE_AUDITING={{ specification.enable_auditing }}
+SCRIPT_AUTO_UNSEAL={{ specification.vault_script_unseal }}
+SECRET_PATH={{ specification.secret_mount_path }}
+KUBERNETES_INTEGRATION={{ specification.kubernetes_integration }}
+LOG_DIR={{ specification.paths.logs }}
```

Comment:
```suggestion
VAULT_INSTALL_PATH="{{ specification.vault_home_dir }}"
ENABLE_AUDITING={{ specification.enable_vault_audit_logs }}
SCRIPT_AUTO_UNSEAL={{ not specification.vault_unsealing.manual }}
SECRET_PATH={{ specification.secret_mount_path }}
KUBERNETES_INTEGRATION={{ specification.enable_kubernetes_auth_method }}
LOG_DIR=/var/logs/vault
```

Reply:
enable_kubernetes_auth_method - I will add additional setup to functions enabled by this parameter, so I will use kubernetes_integration instead